homeassistant/components/switchbot/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "switchbot",
   "name": "SwitchBot",
   "documentation": "https://www.home-assistant.io/integrations/switchbot",
-  "requirements": ["PySwitchbot==0.18.12"],
+  "requirements": ["PySwitchbot==0.18.13"],
```

Comment:
13 or 14?

Reply:
Thanks for catching that. Guess I was a bit tired and forgot to push!